opentracing/tracecontext.go

```diff
@@ -14,55 +14,47 @@ import (
 // TraceContexts are sufficient to propagate the, well, *context* of a
 // particular trace between processes.
 //
-// TraceContext also support a simple string map of "trace tags". These trace
-// tags are special in that they are propagated *in-band*, presumably alongside
-// application data. See the documentation for SetTraceTag() for more details
-// and some important caveats.
+// TraceContext also support a simple string map of "trace attributes". These
```

Comment:
The word trace hints at scope (if coming from zipkin/dapper). Is it assumed that these attributes will add or replace regardless of span depth?